packages/block-template/package.json

```
@@ -29,7 +29,7 @@
     "schema": "typescript-json-schema tsconfig.json AppProps --required true --out dist/block-schema.json"
   },
   "dependencies": {
-    "blockprotocol": "0.0.1"
+    "blockprotocol": "*"
```

Not sure we want to make this requirement so loose. A specific version of a template conceptually depends on a specific version of `blockprotocol`. We can still use `*` in `site/package.json`, but I’d go for 0.0.4 everywhere for ‘symmetry’.

Tools like Lerna automate version bumps like this, so this manual edits are temporary. 

```suggestion
    "blockprotocol": "0.0.4"
```

---

This change (to `"*"`) was in response to your previous comment:
> I guess we need to update dependency in site/package.json & packages/block-template/package.json. We should not depend on an external BP version.

This returns it to depending on an external version, which does not exist. Is the intention that we do in fact go back to depending on an external version?

---

Or that we should always have this be the same value as the latest published, but pin it so that when people use it elsewhere they get that version?

---

Actually, maybe it wasn't picking up 0.0.4 from the workspace before because of #124 - will commit this and see what happens